test/cypress/e2e/fixtures/texts.json

```diff
@@ -19,7 +19,7 @@
   "requiredFieldLastName": "Du må fylle ut nytt etternavn",
   "requiredFieldDateFrom": "Du må fylle ut dato for navneendring",
   "requiredFieldFromValue": "Du må fylle ut 1. endre fra",
-  "requiredFieldToValue": "Du må fylle ut 2. endre verdi {0} til ",
+  "requiredFieldToValue": "Du må fylle ut 2. endre verdi  til ",
```

Comment:
Shouldn't this actually render some text in this test?
This would mean that the field is not working properly.
I agree that `{0}` doesn't make much sense either, but this looks like the substitution is not working properly.

Reply:
Here is a screenshot of the test if I fill inn the first textfield and click save: 
![image](https://user-images.githubusercontent.com/43168787/191755759-673a5c18-b74e-4c0e-9171-d45637e67b65.png)

Reply:
I guess the value of the first textfield should actually be in the error message.

Reply:
Yes I can see that the errormessage is correctly updated after clicking "Lagre og lukk". I Can change the test so it will reflect this.